cleanlab/filter.py

```diff
@@ -290,38 +281,43 @@ def get_noise_indices(
 
     num_to_remove_per_class : list of int of length K (# of classes)
       e.g. if K = 3, num_to_remove_per_class = [5, 0, 1] would return
-      the indices of the 5 most likely mislabeled examples in class s = 0,
-      and the most likely mislabeled example in class s = 1.
+      the indices of the 5 most likely mislabeled examples in class labels = 0,
+      and the most likely mislabeled example in class labels = 1.
 
       Note
       ----
-      Only set this parameter if ``prune_method == 'prune_by_class'``
-      You may use with ``prune_method == 'prune_by_noise_rate'``, but
+      Only set this parameter if ``filter_by == 'prune_by_class'``
+      You may use with ``filter_by == 'prune_by_noise_rate'``, but
       if ``num_to_remove_per_class == k``, then either k-1, k, or k+1
       examples may be removed for any class. This is because noise rates
       are floats, and rounding may cause a one-off. If you need exactly
       'k' examples removed from every class, you should use ``'prune_by_class'``
 
-    prune_method : str (default: 'prune_by_noise_rate')
-      Possible Values: 'prune_by_class', 'prune_by_noise_rate', or 'both'.
-      Method used for pruning.
+    filter_by : str (default: 'prune_by_noise_rate')  TODO: change default to cl_off_diag?
+      Possible Values: {'prune_by_class', 'prune_by_noise_rate', 'both',
+                        'confident_learning', 'argmax_not_equal_given_label'}
+      Method used for filtering/pruning out the label issues.
       1. 'prune_by_noise_rate': works by removing examples with
       *high probability* of being mislabeled for every non-diagonal
-      in the prune_counts_matrix (see pruning.py).
+      in the prune_counts_matrix (see filter.py).
       2. 'prune_by_class': works by removing the examples with *smallest
       probability* of belonging to their given class label for every class.
       3. 'both': Finds the examples satisfying (1) AND (2) and
       removes their set conjunction.
+      4. 'confident_learning': Find examples that are confidently labeled as a class
+      that'labels different from their given label while computing the confident joint.
+      5. 'argmax_not_equal_given_label': Find examples where the argmax predictions does not match
```

Comment:
'misclassified' instead of 'argmax_not_equal_given_label'?

Reply:
"the argmax predictions" 
->
"the predicted class (i.e. argmax of the predicted probabilities)"

Reply:
second change made.

for first change, I've changed `argmax_not_equal_given_label` to `predicted_neq_given`. I want to avoid the term `misclassified` because misclassified implies `pred != true label`, where in this case, we mean instead `pred != given label`, which is slightly different.